examples/common/pigweed/rpc_services/Device.h

```diff
@@ -418,6 +419,21 @@ class Device : public pw_rpc::nanopb::Device::Service<Device>
         {
             mCommissionableDataProvider.SetSpake2pVerifier(ByteSpan(request.verifier.bytes, request.verifier.size));
         }
+
+        if (Server::GetInstance().GetCommissioningWindowManager().IsCommissioningWindowOpen())
```

Comment:
Really, this should check for a _basic_ commissioning window being open...

Reply:
Done